bin/openaps-install.sh

```diff
@@ -70,7 +76,7 @@ fi
 sed -i "s/daily/hourly/g" /etc/logrotate.conf
 sed -i "s/#compress/compress/g" /etc/logrotate.conf
 
-curl -s https://raw.githubusercontent.com/openaps/oref0/$BRANCH/bin/openaps-packages.sh | bash -
+curl -s https://raw.githubusercontent.com/sharbi/oref0/patch-1/bin/openaps-packages.sh | bash -
```

Comment:
This kind of change should not be pushed. It's a fine change to make locally for testing purposes, of course.

Reply:
will keep in mind! thanks